Picon.cz2VDR.sh

```diff
@@ -233,6 +233,7 @@ for i in "${!channelsconf[@]}" ; do
     else
       servicename="${vdr_channelname,,[A-Z]}"  # In Kleinbuchstaben (Außer Umlaute)
     fi  # TOLOWER
+    servicename="${servicename/|/:}"           # Kanal mit | im Namen
```

Comment:
I sugest to use 

`servicename="${servicename//|/:}"           # Kanal mit | im Namen`

to replace all '|'

Reply:
Sorry, seen to late the comment. Should I prepare new PR, to resolve this?

Reply:
No. Already changed it. Thank you for the support!